modelcenter/PP-LCNetV2/introduction_en.ipynb

```diff
@@ -0,0 +1,239 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The above command results are as follows:\n",
+    "\n",
+    "class_ids: [8, 7, 86, 82, 83], scores: [0.8859, 0.07156, 0.00588, 0.00047, 0.00034], label_names: ['hen', 'cock', 'partridge', 'ruffed grouse, partridge, Bonasa umbellus', 'prairie chicken, prairie grouse, prairie fowl'], filename: docs/images/inference_deployment/whl_demo.jpg\n",
+    "Predict complete"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 3.2 Training\n",
+    "* PP-LCNetV2 series models are implemented based on PaddleClas. For details of model training, please refer to[Training, Evaluation and Inference](https://github.com/PaddlePaddle/PaddleClas/blob/release/2.5/docs/en/models/PP-LCNetV2_en.md)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 4. Theory"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 4.1 Overall Structure\n",
+    "\n",
+    "The overall structure of the network is shown in the figure below:\n",
+    "\n",
+    "<div align=\"center\">\n",
+    "<img src=\"https://user-images.githubusercontent.com/12560511/200776415-19f7f09d-f8e9-4b64-bbcd-72ac418c23cf.png\"  width = \"80%\"  />\n",
+    "</div>\n",
+    "\n",
+    "### 4.2 Model Details\n",
+    "\n",
+    "PP-LCNetV2 model is optimized on the basis of PP-LCNetV1. It mainly uses the reparameterization strategy to combine the deep convolution of convolution cores of different sizes, and optimize the point convolution and Shortcut, etc.\n",
+    "\n",
+    "#### 4.2.1  Re-parameterization\n",
+    "\n",
+    "The size of the convolution kernel affects the size of the model's receptive field, which affects the model's ability to capture more global or local features. In order to help the model build different scales features, we use 5*5, 3*3 and 1*1 size convolution kernel. The details is shown in the figure below.\n",
+    "\n",
+    "<div align=\"center\">\n",
+    "<img src=\"https://user-images.githubusercontent.com/12560511/201664156-be8f5dc6-5345-436a-962e-079139f378d5.png\"  width = \"80%\"  />\n",
+    "</div>\n",
+    "\n",
+    "#### 4.2.2 PW Conv\n",
+    "\n",
+    "The depth separable convolution usually consists of a layer of DW convolution and a layer of PW convolution to replace the standard convolution. In order to make the depth separable convolution have a stronger fitting ability, we try to use two layers of PW convolution. Meanwhile, in order to control the efficiency of the model, the two-layer PW convolution is set as: The first is to compress the feature map in the channel dimension, and the second is to zoom in to restore the feature map channel, as shown in the figure below. Experiments show that this strategy can significantly improve model performance. Meanwhile, in order to balance the impact on model efficiency, PPLCNetV2 only uses this strategy in Stage4.\n",
```

Comment:
depth separable convolution -> depthwise separable convolution

Reply:
in the channel dimension ->  on the channel dimension

Reply:
The first squeeze the feature map on the channel dimension, and the second expand to restore the feature map channel